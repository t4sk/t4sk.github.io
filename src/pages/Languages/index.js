import React from "react"
import Container from "react-bootstrap/Container"
import Layout from "../../components/Layout"
import SEO from "../../components/SEO"

export function Languages(prop) {
  return (
    <Layout>
      <SEO title="Languages | Smart Contract Programmer" />
      <ul>
        <li>
          <a href="/en">English</a>
        </li>

        <li>
          <a href="/jp">Japanese - 日本語</a>
        </li>
      </ul>
    </Layout>
  )
}

export default Languages
