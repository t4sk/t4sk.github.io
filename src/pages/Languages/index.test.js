import React from "react"
import { shallow } from "enzyme"
import Languages from "./index"

test("it renders", () => {
  const component = shallow(<Languages />)

  expect(component).toMatchSnapshot()
})
