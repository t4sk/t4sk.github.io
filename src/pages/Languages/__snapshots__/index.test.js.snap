// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders 1`] = `
<Layout
  lang="en"
>
  <SEO
    description="Learn how to program smart contracts."
    keywords={
      Array [
        "smart contract",
        "solidity",
        "ethereum",
      ]
    }
    lang="en"
    meta={Array []}
    title="Languages | Smart Contract Programmer"
  />
  <ul>
    <li>
      <a
        href="/en"
      >
        English
      </a>
    </li>
    <li>
      <a
        href="/jp"
      >
        Japanese - 日本語
      </a>
    </li>
  </ul>
</Layout>
`;
